src/Pdf/CakePdf.php

```diff
@@ -738,6 +752,40 @@ public function template($template = false, $layout = null)
     }
 
     /**
+     * Template path
+     *
+     * @param mixed $templatePath The path of the template to use
+     * @return mixed
+     */
+    public function templatePath($templatePath = false)
```

Comment:
Might be, i just used took same approach as rest, where if use null to delete the current value, false to get the current value, and string to set new value - so I did this for consistency.